saltproc/abc.py

```diff
@@ -0,0 +1,163 @@
+from abc import ABC, abstractmethod
```

Comment:
This file should be renamed to avoid a potential namespace conflict with `abc`. Also so that it's clearer to users what lives here.

Be sure to use `git mv abc.py some-other-name.py` to minimize the number of files in the commit.